File: apps/furniture-store-service-server/src/APIs/OrderItem/Dtos/OrderItemCreateInput.cs
namespace FurnitureStoreService.APIs.Dtos;

public class OrderItemCreateInput
{
    public string? Id { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public double? Price { get; set; }

    public int? Quantity { get; set; }

    public Order? Order { get; set; }

    public Product? Product { get; set; }
}

File: apps/furniture-store-service-admin/src/api/orderItem/OrderItemCreateInput.ts
import { ProductWhereUniqueInput } from "../product/ProductWhereUniqueInput";
import { OrderWhereUniqueInput } from "../order/OrderWhereUniqueInput";

export type OrderItemCreateInput = {
  quantity?: number | null;
  product?: ProductWhereUniqueInput | null;
  order?: OrderWhereUniqueInput | null;
};
